cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -1768,7 +1768,7 @@ impl<T: Config> BlockNumberProvider for RelaychainDataProvider<T> {
 			.unwrap_or_else(|| Pallet::<T>::last_relay_block_number())
 	}
 
-	#[cfg(feature = "runtime-benchmarks")]
+	#[cfg(any(feature = "runtime-benchmarks", test))]
```

Comment:
let's add std also, just in case we need to create some testing utility for later reuse
```suggestion
	#[cfg(any(feature = "std", feature = "runtime-benchmarks", test))]
```